src/__test__/Media.test.tsx

```diff
@@ -485,10 +485,12 @@ describe("Media", () => {
       const container = document.createElement("div")
       document.body.appendChild(container)
 
-      mockCurrentDynamicBreakpoint("medium")
-
-      container.innerHTML = ReactDOMServer.renderToString(<App />)
+      const html = ReactDOMServer.renderToString(<App />)
+      expect(html).toMatchSnapshot()
+      mockCurrentDynamicBreakpoint("medium");
```

Comment:
To simulate a real world scenario this needs to be mocked after the server does the rendering